src/node/server/serverPluginAssets.ts

```diff
@@ -1,13 +1,21 @@
 import { ServerPlugin } from '.'
 import { isImportRequest, isStaticAsset } from '../utils'
 
-export const assetPathPlugin: ServerPlugin = ({ app }) => {
-  app.use(async (ctx, next) => {
-    if (isStaticAsset(ctx.path) && isImportRequest(ctx)) {
-      ctx.type = 'js'
-      ctx.body = `export default ${JSON.stringify(ctx.path)}`
-      return
-    }
-    return next()
-  })
+interface AssetPathPluginOptions {
+  include?: (file: string) => boolean
+}
+
+export const createAssetPathPlugin = ({
+  include = isStaticAsset
+}: AssetPathPluginOptions = {}): ServerPlugin => {
+  return ({ app }) => {
+    app.use(async (ctx, next) => {
```

Comment:
The `include` is unncessary, that can come form `ServerPluginContext`.

Reply:
👍 Thanks, fixed in bbc7300